dotcom-rendering/src/lib/dailyArticleCount.test.ts

```diff
@@ -35,7 +35,7 @@ describe('dailyArticleCount', () => {
 	});
 
 	it('returns array of valid daily article counts if they exist', () => {
-		localStorage.setItem(
+		storage.local.set(
 			DailyArticleCountKey,
 			JSON.stringify({ value: validDailyArticleCount }),
```

Comment:
If you move to the `@guardian/libs`, you no longer need to set an object with a value!

```suggestion
			validDailyArticleCount
```

I would, however, consider leaving the tests as-is when doing this PR!

Reply:
great idea! caught a few problems. I've reverted the spec to what's on `main`.